src/components/views/spaces/SpacePanel.tsx

```diff
@@ -337,6 +349,14 @@ const SpacePanel = () => {
         }
     });
 
+    useEffect(() => {
+        try {
+            localStorage.setItem(KEY_PANEL_IS_COLLAPSED, isPanelCollapsed);
+        } catch (error) {
+            console.log(error);
+        }
```

Comment:
what are we expecting to fail with these localstorage calls?

Reply:
I read through some blog posts and this catch behavior was common. So, I implemented it. Now, that you asked about, it doesn't seems necessary. I'm going to write the cypress test and if everything is ok, I think I will remove this try/catch block. What do you think?

Reply:
That seems like a good way forward, yea. There's a chance that localstorage will break, but also we don't guard it elsewhere.

Not that it really helps the safety here, we could also move the localstorage key over to the Settings system as a device-only setting, making it safer in the future (if we ever add safety).